.github/workflows/golangci-lint.yml

```diff
@@ -0,0 +1,21 @@
+name: golangci-lint
+on:
+  pull_request:
```

Comment:
https://docs.github.com/en/actions/reference/events-that-trigger-workflows#pull_request
I think you need to supply the types if you are using this syntax
```
on:
  pull_request:
    types: [assigned, opened, synchronize, reopened]
```

or use this syntax
```
on: [pull_request]
```

This is probably what is preventing this GH action from running on this PR.

Reply:
Fixed in another way, the syntax above was taken from an example